.github/workflows/c++-code-formatting.yml

```diff
@@ -342,8 +342,10 @@ jobs:
                     "Remove the trailing spaces at the end of the line.")
             }
             match($0, /\t+/) {
-              error("Tab characters found",
-                    "Indent code using spaces instead of tabs.")
+              if (FILENAME !~ /[Mm]akefile/) {
```

Comment:
I would do it the other way around. I would only complain about tabs if they are found in .c / .cxx files. There is plenty of cases where having tabs might actually be justified.